pkg/kubelet/volumes.go

```diff
@@ -305,10 +297,7 @@ func (kl *Kubelet) newVolumeUnmounterFromPlugins(kind string, name string, podUI
 		// TODO: Maybe we should launch a cleanup of this dir?
 		return nil, fmt.Errorf("can't use volume plugins for %s/%s: %v", podUID, kind, err)
 	}
-	if plugin == nil {
```

Comment:
This nil check has to go back in now, otherwise you will have a nil pointer when you call `plugin.NewUnmounter`